src/routes/blog/spyder-6-daniel/+page.md

```diff
@@ -0,0 +1,102 @@
+However, the one major exception to that in Spyder 5 was the Editor, which wasn't migrated yet due to the high level of complexity and the further new features that would need to be added to the plugin API.
+
+Migrating the Editor to use the modular plugin API was the final piece of the puzzle to complete the API migration making Spyder a fully modular, pluggable application, avoid having to maintain two duplicate APIs and allow interoperability with other plugins, such as those being developed for remote development in Spyder 6.
+Therefore, along with its related API additions and improvements, it also was a prerequisite for fully implementing our new remote development architecture.
+Therefore, we prioritized it for Spyder 6, with Daniel taking the lead on implementing it!
+
+Given the size of the task, Daniel split it into a number of steps, each corresponding to a GitHub Pull Request.
+First, [he split](https://github.com/spyder-ide/spyder/pull/20893) the ``CodeEditor``, the main widget for editing code, into a component in charge of code completion and linting, and another one in charge of editing.
+This helped separate those concerns and improve improve overall modularity and maintainability.
+Then, [he refactored the ``EditorStack`` code](https://github.com/spyder-ide/spyder/pull/21194), the widget in charge of displaying multiple tabbed ``CodeEditor`` views, to facilitate the migration.
+Finally, he migrated the Editor itself to the new API in two steps: an [initial migration](https://github.com/spyder-ide/spyder/pull/21353), and a [followup cleanup](https://github.com/spyder-ide/spyder/pull/22005).
+This last step alone took six months of hard work, but it will help to address a lot of technical debt accumulated throughout the years.<br><br>
+
+
+![The Spyder remote kernel status dialog showing a successful connection, next to the IPython Console showing a kernel error](remote-kernel-issue-windows.png "Daniel helped find and solve remote development issues like this one")
+
+## Remote development architecture quality testing
+
+Once the Editor migration was moving along, helped review, test and QA the new remote development architecture.
+As part of that, he also developed a number of fixes and improvements to address the issues he found.
+From that, he [implemented UI/UX improvements](https://github.com/spyder-ide/spyder/pull/22303) for the graphical components used to create a remote connection, based on the issues he found.
+
+Furthermore, during his checks of remote connections were working on Windows, he discovered several errors with when running async code, which [he fixed](https://github.com/spyder-ide/spyder/pull/22137).
+Finally, he also encountered problems when trying to tunnel ports through SSH, which he [also solved](https://github.com/spyder-ide/spyder/pull/22223](https://github.com/spyder-ide/spyder/pull/22223).
+This all helped make our remote development architecture much more stable and usable.
```

Comment:
Probably here a link to the more detailed blog post about the remote development implementation from HEndrick could makes sense. So something like:

```suggestion
This all helped make our remote development architecture much more stable and usable. If you want to know more about the remote development work and Henrick's experience while working on it give a check to [his blog post!](https://www.spyder-ide.org/blog/spyder-6-remote-development/)
```